cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -23,6 +23,143 @@ Cask Data Application Platform Release Notes
+=============================================================
+
+New Features
+------------
+- `CDAP-9999 <https://issues.cask.co/browse/CDAP-9999>`__ - Added...
+
+
+Improvements
+------------
+- `CDAP-9999 <https://issues.cask.co/browse/CDAP-9999>`__ - Added...
+
+
+Bug Fixes
+---------
+- `CDAP-9999 <https://issues.cask.co/browse/CDAP-9999>`__ - Fixed a problem with...
+
+
+API Changes
+-----------
+- `CDAP-9999 <https://issues.cask.co/browse/CDAP-9999>`__ - Added...
+
+
+Deprecated and Removed Features
+-------------------------------
+- See the :ref:`CDAP 3.4.0 Javadocs <javadocs>` for a list of deprecated and removed APIs.
+- `CDAP-9999 <https://issues.cask.co/browse/CDAP-9999>`__ - Added...
+
+
+.. _known-issues-340:
+
+Known Issues
+------------
+
+- After upgrading CDAP from a pre-3.0 version, any unprocessed metrics data in Kafka will
+  be lost and *WARN* log messages will be logged that tell about the inability to process
+  old data in the old format.
+
+- `CDAP-797 <https://issues.cask.co/browse/CDAP-797>`__ - 
+  When running secure Hadoop clusters, debug logs from MapReduce programs are not
+  available.
+
+- `CDAP-1007 <https://issues.cask.co/browse/CDAP-1007>`__ -
+  If the Hive Metastore is restarted while the CDAP Explore Service is running, the
+  Explore Service remains alive, but becomes unusable. To correct, restart the CDAP Master
+  |---| which will restart all services |---| as described under "Starting CDAP Services"
+  for your particular Hadoop distribution in the :ref:`Installation documentation <installation-index>`.
+
+- `CDAP-1587 <https://issues.cask.co/browse/CDAP-1587>`__ - 
+  User datasets with names starting with ``"system"`` can potentially cause conflicts.
+
+- `CDAP-2632 <https://issues.cask.co/browse/CDAP-2632>`__ -
+  The application in the `cdap-kafka-ingest-guide 
+  <https://github.com/cdap-guides/cdap-kafka-ingest-guide/tree/release/cdap-3.0-compatible>`__ 
+  does not run on Ubuntu 14.x as of CDAP 3.0.x.
+
+- `CDAP-2721 <https://issues.cask.co/browse/CDAP-2721>`__ -
+  Metrics for :ref:`TimePartitionedFileSets <datasets-timepartitioned-fileset>` can show 
+  zero values even if there is data present.
+  
+- `CDAP-2831 <https://issues.cask.co/browse/CDAP-2831>`__ -
+  A workflow that is scheduled by time will not be run between the failure of the primary
+  master and the time that the secondary takes over. This scheduled run will not be
+  triggered at all. There is no warnings or messages about the missed run of the
+  workflow.
+
+- `CDAP-2945 <https://issues.cask.co/browse/CDAP-2945>`__ -
+  There is a problem in the PartitionedFileSet causing MapReduce to fail if the input
+  partition filter does not match any partitions.
+
+- `CDAP-3000 <https://issues.cask.co/browse/CDAP-3000>`__ -
+  There is a problem with the Workflow token being in an inconsistent state for nodes in a
+  fork while the fork is still running. It becomes consistent after the join.
```

Comment:
Also, I don't like how we frame this by starting with "there is a problem". We should just say the issue upfront and not qualify it. Same is true in other places.